notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -917,6 +917,55 @@ what if I was trapped inside a game [Ash discovered that the only way to exit th
 the legend of King Arthur [A priest recanted the story of how a dying King Arthur was taken by ship to the isle of Avalon to see the fairy Morgan le Fay.]
 
 
+movie: Evolution (2001)
+=======================
+
+:: Title
+Evolution
+
+:: Description
+Evolution is a 2001 American comic science fiction film directed by Ivan
+Reitman.
+
+Synopsis: A pair of college professors investigate a meteorite crash in
+Arizona. They discover that the meteorite harbors extraterrestrial lifeforms,
+which are evolving very quickly into large, diverse and outlandish creatures.
+
+:: Date
+2001-06-08
+
+:: References
+https://en.wikipedia.org/wiki/Evolution_(2001_film)
+
+:: Ratings
+2 <paul>
+
+:: Choice Themes
+invasive life form from outer space [The main novelty of the film is that some alien microbes hitched a ride to the Arizona desert on a meteor, began to evolve at an accelerated rate, and soon thereafter gave rise to complex creatures that seemed destined to overrun the entire Earth.]
+
+:: Major Themes
+the scientist profession [The story follows the local college science professors Ira Kane and Harry Block as they study newly discovered, fast-evolving extraterrestrial microbes that had hitched a ride to Earth on a meteor.]
+super accelerated Darwinian evolution [The extraterrestrial microbes evolved into complex dinosaur-like creatures in just a handful of days.]
+romantic love [The story built up to Ira and Allison falling in love in the end.]
+
+:: Minor Themes
+what it is like in university [The viewer is shown various stereotypical community college happenings: An apathetic Professor Ira Kane gave A's to nearly everyone in his biology class on their biology research projects, there was a wild college party, etc.]
+genetics [Ira discovered that the extraterrestrial life was nitrogen-based and had DNA that consisted of ten base pairs.]
+geology [Professor Harry Block made a point that he was a representative of the U.S. Geological Survey when he went to sample a recently crashed meteor.]
+brother and brother [The brothers Deke and Danny Donald were two of the dumbest students in Dr. Ira Kane's biology class.]
+biology [Ira discovered that the extraterrestrial life forms reproduced asexually via mitosis.]
```

Comment:
so was it "sexually distinguished being"?

Reply:
I see we haven't quite defined the "being" in [speculative being](https://themeontology.org/theme.php?name=speculative%20being). Do we want things like asexually reproducing dinosaurs to be beings?

Reply:
I think you've been using speculative being broadly, which is fine. I don't know about the asexually reproducing beings - i doubt we want to use it for bacteria, amorphous blobs etc. Maybe it is only interesting for things that resemble earth life and therefore makes us think "hmm, what would earth/society be like if things/we reproduced without sex?" If that is the case we could change it into a speculative society kind of of theme. Much like single gender society, come to think of it.

Reply:
I checked and all the usages of **sexually distinguished being** and its descendants pertain to societies of beings, which it seems to me in the interesting aspect here. I'm inclined to let the **biology** theme cover the asexual reproduction aspect of this story for the now. Then, down the road, review the **sexually distinguished being** usages with a view to identifying which correspond to novel societies that we might not have covered with other themes.